app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,10 @@
+from app.players import player
```

Comment:
it's better to import class explicitly so you don't have to use dot notation below to access it. Check your other imports throughout the project and adjust where necessary